docs/doxygen/pages.txt

```diff
@@ -29,23 +29,23 @@ The configurations for memory estimation are defined <a href="https://github.com
     </tr>
     <tr>
         <td>core_mqtt.c</td>
-        <td>3.3K</td>
-        <td>2.9K</td>
+        <td>6.9K</td>
```

Comment:
This can be 6.8K as `6.8 * 1024 = 6963.2` is still larger than the `6851` value obtained from your calculations

Reply:
6.7K works too :)